### PR TITLE
Bug 1873623: Revert of add label changes in #5998

### PR DESCRIPTION
**Pull request by andrewballantyne**
Revert of partial changes in #5998 (returning the add labels).

https://issues.redhat.com/browse/ODC-4646

## Screenshots

Screen Shot 2020-08-28 at 3 09 36 PM
Screen Shot 2020-08-28 at 3 11 32 PM
Screen Shot 2020-08-28 at 3 11 39 PM
Screen Shot 2020-08-28 at 3 12 36 PM
Screen Shot 2020-08-28 at 3 13 33 PM
Screen Shot 2020-08-28 at 3 13 46 PM
Screen Shot 2020-08-28 at 3 14 34 PM
Screen Shot 2020-08-28 at 3 15 17 PM
Screen Shot 2020-08-28 at 3 15 38 PM
Screen Shot 2020-08-28 at 3 16 05 PM
Screen Shot 2020-08-28 at 3 16 41 PM

FYI @bgliwa01 / @beaumorley

/cc @debsmita1

### CODE REVIEW

File: frontend/packages/console-shared/src/components/formik-fields/multi-column-field/MultiColumnFieldFooter.tsx [hunk 1]
```diff
@@ -20,7 +20,7 @@ const MultiColumnFieldFooter: React.FC<MultiColumnFieldHeader> = ({
     icon={<PlusCircleIcon />}
     isInline
   >
-    {addLabel || 'Add'}
+    {addLabel || 'Add values'}
```
Comment (debsmita1) [inline frontend/packages/console-shared/src/components/formik-fields/multi-column-field/MultiColumnFieldFooter.tsx:23]: how about `Add value` ?

File: frontend/packages/dev-console/src/components/health-checks/RequestTypeForms.tsx [hunk 2]
```diff
@@ -104,6 +105,7 @@ export const CommandRequestTypeForm: React.FC<RequestTypeFormProps> = ({ probeTy
     <TextColumnField
       name={`healthChecks.${probeType}.data.exec.command`}
       label="Command"
+      addLabel="Add command"
```
Comment (debsmita1) [inline frontend/packages/dev-console/src/components/health-checks/RequestTypeForms.tsx:108]: should we go with **the first letter of every word** in caps or **the first letter of the text**  in caps? better to follow a single pattern

File: frontend/packages/dev-console/src/components/pipelines/pipeline-builder/task-sidebar/temp-utils.tsx [hunk 3]
```diff
@@ -87,6 +87,7 @@ export const ArrayParam: React.FC<ParameterProps> = (props) => {
         {description}
       </p>
       <MultiColumnFieldFooter
+        addLabel="Add another value"
```
Comment (debsmita1) [inline frontend/packages/dev-console/src/components/pipelines/pipeline-builder/task-sidebar/temp-utils.tsx:90]: how about `add value` ?

File: frontend/packages/console-shared/src/components/formik-fields/multi-column-field/MultiColumnFieldFooter.tsx (hunk 1 above)
Comment (andrewballantyne) [reply frontend/packages/console-shared/src/components/formik-fields/multi-column-field/MultiColumnFieldFooter.tsx:23]: I reverted the change where you [removed this value](https://github.com/openshift/console/pull/5998/files#diff-d4751c6c009a3bb5e09991b50a9d98afL23). I didn't customize the text.

File: frontend/packages/dev-console/src/components/health-checks/RequestTypeForms.tsx (hunk 2 above)
Comment (andrewballantyne) [reply frontend/packages/dev-console/src/components/health-checks/RequestTypeForms.tsx:108]: Again, same thing - just a revert of what it was before you [removed it](https://github.com/openshift/console/pull/5998/files#diff-4349b9c69e6160a418fa47e212c28731L108).

File: frontend/packages/dev-console/src/components/pipelines/pipeline-builder/task-sidebar/temp-utils.tsx (hunk 3 above)
Comment (andrewballantyne) [reply frontend/packages/dev-console/src/components/pipelines/pipeline-builder/task-sidebar/temp-utils.tsx:90]: Same thing here, the [value before it was removed](https://github.com/openshift/console/pull/5998/files#diff-6133e47a6edec3216bd17358565eef74L88).

**Comment by andrewballantyne** [comment]
@debsmita1 We have a follow up ticket to cleanup the labels.
https://issues.redhat.com/browse/ODC-4700

**Review by rohitkrai03 — APPROVED** [review]
/approve

**Pull request merged**